src/js/osweb/plugins/touch_response.js

```diff
@@ -52,23 +52,20 @@ export default class TouchResponse extends MouseResponse {
     this.experiment.vars.cursor_x = retval.event.clientX
     this.experiment.vars.cursor_y = retval.event.clientY
 
-    var rect = this._runner._renderer.view.getBoundingClientRect()
-    if (this.experiment.vars.uniform_coordinates === 'yes') {
-      this._x = retval.event.clientX - rect.left
-      this._y = retval.event.clientY - rect.top
-      // this._x = pRetval.event.clientX - rect.left + (this.experiment.vars.width / 2);
-      // this._y = pRetval.event.clientY - rect.top + (this.experiment.vars.height / 2);
-    } else {
-      this._x = retval.event.clientX - rect.left
-      this._y = retval.event.clientY - rect.top
-    }
+    const rect = this._runner._renderer.view.getBoundingClientRect()
+    this._x = retval.event.clientX - rect.left
+    this._y = retval.event.clientY - rect.top
+    // if (this.experiment.vars.uniform_coordinates === 'yes') {

```

Comment:
@smathot Do you know what this segment represents? Is the uniform_coordinates system relevant for osweb at all? If not, I think this can be removed completely.

Reply:
Right, that's a legacy from back when OpenSesame took the top-left as 0, 0 and sometimes the center. But that's never been applicable to OSWeb. So you're right: It can go. I cleaned it up already.